### PR TITLE
View list of items

### PR DESCRIPTION
**Pull request by pearllaw**

### CODE REVIEW

**Comment by pearllaw** [comment]
issue 1

File: main.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  $row.classList.add('row')
+  var $rowTwo = document.createElement('div')
+  $rowTwo.classList.add('row')
+  var equalWidth = document.createElement('div')
+  equalWidth.classList.add('w-100')
+  $container.appendChild($heading)
+  $container.appendChild($row)
+  $container.appendChild($rowTwo)
+
+  for (var i = 0; i < 4; i++) {
```
Comment (thebearingedge) [inline main.js:136]: You can put all of the columns in one `$row`. Use one or more [column classes](https://getbootstrap.com/docs/4.1/layout/grid/#mix-and-match) on each `$col`. Bootstrap uses media queries to automatically move extra columns down the page.
https://getbootstrap.com/docs/4.1/layout/grid/#mix-and-match

File: main.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    var $col = document.createElement('div')
+    $col.classList.add('col')
+    var $item = renderItem(catalog[i])
+    $col.appendChild($item)
+    $rowTwo.appendChild($col)
+  }
+  return $container
+}
+
+var $items = app.catalog.items
```
Comment (thebearingedge) [inline main.js:154]: Instead of adding these two lines, just call your `renderApp` function to show the catalog.

File: your-project/.eslintrc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (thebearingedge) [inline your-project/.eslintrc:1]: Please move everything from `your-project/` into the root of your project directory. Then run `npm install` again.

File: your-project/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# your-project
```
Comment (thebearingedge) [inline your-project/README.md:1]: Please change this to `# jamazon`.

File: main.js [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+document.body.appendChild(renderCatalog($items))
+
+function renderApp(state) {
```
Comment (thebearingedge) [inline main.js:157]: This function will take the whole `app` object.

File: main.js [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+  var $view = document.querySelector('[data-view="catalog"]')
+  $view.appendChild(renderCatalog(app))
```
Comment (thebearingedge) [inline main.js:159]: `renderCatalog` should only be passed the `app.catalog.items`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice! I have a few notes. Also, please put the screenshot or gif in the Pull Request instead of the Issue.

**Pull request closed without merging**